### DISCUSSION TITLE
Multiple Email Addresses, Automated Follow-Up Emails and Overdue Invoice Management

### DISCUSSION THREAD

**Discussion by Mkopl360** [InvoiceShelf/InvoiceShelf#229] [Ideas]
**Feature Idea: Enhanced Email and Invoice Management**  

1. **Multiple Email Addresses**:  
   - Allow adding multiple email addresses to a customer account.  
   - When sending an email, it will automatically be sent to all listed email addresses.  

2. **Automated Follow-Up Emails**:  
   - Implement an automated system to send follow-up emails for invoices:  
     - **7 days before the due date**  
     - **3 days before the due date**  
     - **On the due date**  
     - **After the due date**  
     - **3 days after the due date**  

3. **Overdue Invoice Management**:  
   - Categorize invoices as overdue once they pass the due date.  
   - Provide a dedicated page or filter option to easily view and manage overdue invoices.

**Comment by yasdanibrenes** [comment]
I would also like to add the feature with the option from settings that we can add/set a Cc "carbon copy" email address in case we also would like to have a record in our email inbox of the invoices/estimates/payments emails that are sent out

**Comment by mpolch** [comment]
I need here exatly 12 month late. The current option yearly is starting always from 1st. of January.

**Reply by rihards-simanovics** [reply]
> I need here exatly 12 month late. The current option yearly is starting always from 1st. of January.

Twelve years late? Sounds like a lawsuit by that point 😳.
